cmd/crio/main.go

```diff
@@ -430,6 +431,10 @@ func main() {
 			return fmt.Errorf("invalid --runtime value %q", err)
 		}
 
+		if err := os.MkdirAll(filepath.Dir(config.Listen), 0755); err != nil {
```

Comment:
I'd rather use `0777` and rely on the caller's umask for further restriction.  That way folks who want to provide write access to their group (or, if they're crazy, to everyone) can do so.

Reply:
Really 0777 and not 0775?  I start twitching whenever I see 0777.

Reply:
> Really 0777 and not 0775? I start twitching whenever I see 0777.

Really 0777.  You only need to get twitchy if your umask is empty ;).

```console
$ umask 
0022
$ cat test.go
package main
import "os"
func main() {
  os.Mkdir("testing", 0777)
}
$ stat -c '%a' testing
```

You'll only get 0777 out if you happen to run an empty umask, which would be a bit extreme, but is the caller's decision.